### ISSUE TITLE
replace deprecated `ioutil` package functions with latest advised 'io` and `os` package replacements

### ISSUE THREAD

**Issue by mrutkows** [CycloneDX/sbom-utility#39]
With the advent of v1.19, we now have a mature replacement for all needed functions from the deprecated `ioutil` package (as of v1.16).  

For example, the utility currently uses `ioutil.ReadAll` which would be replaced by `io.ReadAll`.

For a more complete set of package/function mappings see this answer: https://stackoverflow.com/questions/75206234/for-go-ioutil-readall-ioutil-readfile-ioutil-readdir-deprecated

**Comment by ajistrying** [comment]
Hello there! Is this project open to a golang newbie, if so I'd love to take this one as a first issue and continue to contribute as I learn more 🙂

**Comment by mrutkows** [comment]
@ajistrying I would welcome any help with the "good first issue" tag and offer time to help, discuss approaches (if not obvious) and provided review/comment on draft PRs. 

Please know I have changed a couple of these as part of PR: https://github.com/CycloneDX/sbom-utility/pull/40
but have made no attempt at universally replacing all occurrences.

**Comment by ajistrying** [comment]
@mrutkows No worries and thank you! What I can do is start on the replacements and wait for that PR to get merged in to incorporate it 👍

**Comment by mrutkows** [comment]
This file: cmd/license_policy_config.go
needs refactoring

**Comment by mrutkows** [comment]
@ajistrying was wondering if you have time to do this work in the next day or so as I want to rush a release in order to support the latest CycloneDX v1.5 release which happened yesterday.  If you are too busy, I may fix the remaining deprecated occurrences just to get them into this release.  Sorry, to ask this of you.

**Comment by ajistrying** [comment]
@mrutkows Yes I definitely can get this done by tomorrow. The start of this week has been pretty busy so apologies!

**Comment by mrutkows** [comment]
@ajistrying Thanks. I will keep looking for your PR to be submitted and if it looks good create a release after merging it.

**Comment by ajistrying** [comment]
Just signing off for work, will push up something in the next hour or so!

**Comment by ajistrying** [comment]
@mrutkows #44 is up! Would love some insight on the automated comments from the sonatype-lift bot on the potential issues it highlighted.

**Comment by mrutkows** [comment]
Fixed by: https://github.com/CycloneDX/sbom-utility/pull/44
